docs/reference-guide/query-modules/implement-custom-query-modules/api/python-api.md

```diff
@@ -1352,7 +1352,7 @@ def create_edge(from_vertex: Vertex, to_vertex: Vertex,
                 edge_type: EdgeType) -> None
 ```
 
-Create an edge.
+Create an edge. When the first parameter is projected graph, it will create new directed edge with specified label only if both vertices are part of projected graph.
```

Comment:
```suggestion
Create an edge. When the first parameter is a projected graph, it will create a new directed edge with a specified label only if both vertices are a part of the projected graph.
```